NEWS.md

```diff
@@ -1,3 +1,11 @@
+# ggExtra 0.6.1.9000
+
+#### Refactoring of `ggMarginal`:
+- Several of the stages/tasks that `ggMaringal` completes to create the final plot are now broken into (more) helper functions
+- The accession of the `panel_scales` object is now dependent on the `ggplot2` version used (see `getPanelScale` function)
+- Removed code that added y labels to marginal plot to deal with long y labels as well as spacing issues
```

Comment:
Can you add a reference to the actual code so it's easy to find in the future https://github.com/daattali/ggExtra/blob/6c4923c2ad2e700226cdcdc9666de513d6ae3a41/R/ggMarginal.R#L186-L201